tests/fi/_fixtures.yaml

```diff
@@ -1,36 +1,36 @@
 language: "fi"
 areas:
-  - name: "keittiö"
-    id: "kitchen"
-  - name: "olohuone"
-    id: "living_room"
-  - name: "makuuhuone"
-    id: "bedroom"
-  - name: "autotalli"
-    id: "garage"
-  - name: "työhuone"
-    id: "office"
-  - name: "eteinen"
-    id: "hall"
-  - name: "sauna"
+  - name: "(keittiö | keittiön)"
```

Comment:
Like we discussed, this is one option, but it would require all the Finnish home assistant users to add conjugations to all of their areas and devices as aliases. These two formats are probably not enough, the actual number is higher.

After I saw you use this approach, I also considered, but I'd recommend we do not add these conjugations/declinations here until the issue https://github.com/home-assistant/intents/issues/907 is solved one way or another. The reason being, that it puts the pressure on users to add all the formats.